script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
```

Comment:
Could you add some comments with a basic explanation of what things this code is checking? Collections are pretty confusing to me, I think this would help anyone going forward to understand this check better.